src/network/networkd-manager.c

```diff
@@ -90,25 +90,25 @@ static int on_bus_retry(sd_event_source *s, usec_t usec, void *userdata) {
 
         m->bus_retry_event_source = sd_event_source_unref(m->bus_retry_event_source);
 
-        manager_connect_bus(m);
-
-        return 0;
+        return manager_connect_bus(m);
 }
 
 static int manager_reset_all(Manager *m) {
         Link *link;
         Iterator i;
-        int r;
+        int k, r = 0;
 
         assert(m);
 
         HASHMAP_FOREACH(link, m->links, i) {
-                r = link_carrier_reset(link);
-                if (r < 0)
+                k = link_carrier_reset(link);
+                if (k < 0) {
                         log_link_warning_errno(link, r, "Could not reset carrier: %m");
+                        r = k;
+                }
```

Comment:
I can't figure what it exactly have issue with the crash ?

Reply:
No it does not related. The relevant commit is not related to any issue. I thought that in principle we should propagate errors. Do you think that in this case we should not propagate it?

Reply:
Many returned values, especially errors are ignored in networkd. So, I want to fix them. Of course, some of them should be ignored, but in such case, we should cast them to `(void)` for readability... Right?